scripts/build/rollup.config.js

```diff
@@ -54,6 +55,39 @@ export const umd = {
   ],
 };
 
+const updateLibPrefix = (isUndo) => {
+  const path = 'src/lib-prefix.ts'
+  if (!fs.existsSync(path)) {
+    // Supported in rollup 4, we're currently rollup 2
+    // this.error(`File not found: ${path}`);
```

Comment:
Minor, do we want to keep this logging comment?

Reply:
I'd like to keep it here so that when we migrate to rollup 4, we can just uncomment this line. wdyt?